### DISCUSSION TITLE
Entire page deopted into client-side rendering

### DISCUSSION THREAD

**Discussion by Kofi-Emma** [medusajs/medusa#6570] [Q&A]
I am trying to push my medusa frontend to vercel but I get this error when I try

Generating static pages (0/15)  [=   ] ⚠ Entire page /checkout deopted into client-side rendering. https://nextjs.org/docs/messages/deopted-into-client-rendering /checkout
 ⚠ Entire page /account/login deopted into client-side rendering. https://nextjs.org/docs/messages/deopted-into-client-rendering /account/login
 ⚠ Entire page /account deopted into client-side rendering. https://nextjs.org/docs/messages/deopted-into-client-rendering /account
 ⚠ Entire page /account/addresses deopted into client-side rendering. https://nextjs.org/docs/messages/deopted-into-client-rendering /account/addresses
 ⚠ Entire page /404 deopted into client-side rendering. https://nextjs.org/docs/messages/deopted-into-client-rendering /404
 ⚠ Entire page /account/orders deopted into client-side rendering. https://nextjs.org/docs/messages/deopted-into-client-rendering /account/orders
 ⚠ Entire page /account/profile deopted into client-side rendering. https://nextjs.org/docs/messages/deopted-into-client-rendering /account/profile
   Generating static pages (6/15)  [==  ] 
 ⚠ Entire page /cart deopted into client-side rendering. https://nextjs.org/docs/messages/deopted-into-client-rendering /cart
   Generating static pages (7/15)  [=== ]

What am I doing wrong?

**Comment by inacio000** [comment]
I found a guy with the same problem, and I think he solved It...

 You may check the post on the link bellow.
 
 https://www.reddit.com/r/nextjs/comments/13h5xvf/why_is_deopting_to_client_side_rendering/
